src/Middleware/CORS/src/Infrastructure/DefaultCorsPolicyProvider.cs

```diff
@@ -34,7 +34,7 @@ public Task<CorsPolicy> GetPolicyAsync(HttpContext context, string policyName)
             policyName ??= _options.DefaultPolicyName;
             if (_options.PolicyMap.TryGetValue(policyName, out var result))
             {
-                return result.policyTask;
+                return result.policyTask!;
```

Comment:
💡 An [`AsNullable`](https://github.com/dotnet/roslyn/blob/adaa46d777e4be3c70c5f7455e39b58b0db0fb89/src/Workspaces/SharedUtilitiesAndExtensions/Compiler/Core/Utilities/SpecializedTasks.cs#L27-L28) extension method helps avoid accidental misinterpretation of this suppression.

```suggestion
                return result.policyTask.AsNullable();
```